docs/03-pages/01-building-your-application/07-deploying/04-ci-build-caching.mdx

```diff
@@ -126,3 +126,12 @@ Using Azure Pipelines' [Cache task](https://docs.microsoft.com/en-us/azure/devop
     key: next | $(Agent.OS) | yarn.lock
     path: '$(System.DefaultWorkingDirectory)/.next/cache'
 ```
+
+### Monorepos
+
+In a monorepo, you might have multiple Next.js apps in subdirectories. To ensure caching works for all of these apps, modify your cache path as follows:
+
+`path: ${{ github.workspace }}/.next/cache`
+
+to:
+`path: ${{ github.workspace }}/apps/*/.next/cache`
```

Comment:
Should this be nested under GitHub Actions?

Reply:
Do you prefer to add a monorepo section for each git provider?
It will double the content size, although I do think it is a good decision.

Then need check if the same trick works for all git providers such as Azure, GitLab, and so on. 
I didn't succeed to configure this for our monorepo at azure devops yet, I'll give it another try.